docs/messaging.rst

```diff
@@ -547,6 +547,14 @@ Message type: ``inspect_request``::
     ``name`` key replaced with ``code`` and ``cursor_pos``,
     moving the lexing responsibility to the kernel.
 
+.. versionchanged:: 5.2
+
+    Due to a widespread bug in many frontends, ``cursor_pos``
+    in versions prior to 5.2 is ambiguous in the presence of astral-plane.
```

Comment:
missing a noun:
```
in the presence of "astral-plane" characters
```
(added scare quotes around "astral-plane" since this is a slang term)